mempool/cat/store.go

```diff
@@ -142,14 +142,17 @@ func (s *store) getTxsBelowPriority(priority int64) ([]*wrappedTx, int64) {
 
 // purgeExpiredTxs removes all transactions that are older than the given height
 // and time. Returns the amount of transactions that were removed
-func (s *store) purgeExpiredTxs(expirationHeight int64, expirationAge time.Time) int {
+func (s *store) purgeExpiredTxs(expirationHeight int64, expirationAge time.Time, evictedTxCache *LRUTxCache) int {
```

Comment:
[no change needed] it seems like `purge` could be renamed to `evict` here. IMO it's easier to understand what's happening if we use consistent names for consistent things.

In other words, it isn't immediately obvious to me that purging a transaction should add it to the evicted tx cache. However, evicting a transaction seems like it should add the tx to the evicted tx cache.